src/dataflow/src/render/upsert.rs

```diff
@@ -462,7 +487,9 @@ where
                                                 res.map(|v| {
                                                     rehydrate(
                                                         &upsert_envelope.key_indices,
-                                                        &decoded_key,
+                                                        // The value is never `Ok`
```

Comment:
This is just me wondering: do we really guarantee that? They are decoded from separate fields of the original Kafka message. Please feel free to ignore, though.